server_addon/maya/client/ayon_maya/plugins/publish/extract_ornatrix_rig.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""Extract Ornatrix rig."""
+
+import os
+import json
+
+from ayon_maya.api import lib
+from ayon_maya.api import plugin
+from maya import cmds
+
+
+class ExtractOxRig(plugin.MayaExtractorPlugin):
+    """Extract the Ornatrix rig to a Maya Scene and write the Ornatrix rig data."""
+
+    label = "Extract Ornatrix Rig"
+    families = ["oxrig"]
+    scene_type = "ma"
+
+    def process(self, instance):
+        """Plugin entry point."""
+        cmds.loadPlugin("Ornatrix.mll", quiet=True)
```

Comment:
Just thought of this now, but I believe the usage of the `.mll` suffix here makes it Windows only since the plug-in files are `.so` on linux/mac? You should be able to load these without the `.mll` suffix to be cross-platform?

Reply:
maybe just doing `cmds.loadPlugin("Ornatrix", quiet=True)`

Reply:
Yes, exactly. 👍